lms/djangoapps/dashboard/tests/test_sysadmin.py

```diff
@@ -120,6 +122,126 @@ def _mkdir(self, path):
         self.addCleanup(shutil.rmtree, path)
 
 
+@unittest.skipUnless(
+    settings.FEATURES.get('ENABLE_SYSADMIN_DASHBOARD'),
+    "ENABLE_SYSADMIN_DASHBOARD not set",
+)
+class TestSysadminEditCourseTabs(SysadminBaseTestCase):
```

Comment:
Hrmmm... this file is starting to get pretty large, nearing 1000 lines.

What are your thoughts about splitting this into its own file? We could do something like `lms/djangoapps/dashboard/tests/test_sysadmin_course_tabs.py` or some such.